en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+# Battery and Powermodule Setup
+
+> **Warning** Before you fly for the first time you should make sure you have a way to monitor your battery state. Not knowing how much fuel is left may likely lead to deep discharge of the used battery which can cause damage to the battery, loss of vehicle control and hence a crash.
+
+> **Note** The battery monitoring related features of PX4 can only be used if you have compatible hardware which is in most cases a power module that measures the battery voltage and the current between battery and vehicle.
+
+## Set the Basic Battery Settings
+
+1. Start *QGroundControl* and connect the vehicle.
+1. Select the **Gear** icon (Vehicle Setup) in the top toolbar and then **Power** in the sidebar.
+
+You are presented with the basic settings that characterize your battery.
+
+### Number of accumulator cells connected in series
+Batteries commonly rely on a chemical process to save and supply energy. The voltage across one atomic battery cell is given by the chemical properties of the battery type.
```

Comment:
OK, so I prefer to have intro about section and simple how to instructions, THEN the explanation of how things work. This allows people to understand what they have to do, before seeing the detail.  So in this case I would lead with the last line you added:

```
This setting sets the number of cells connected in series PX4 should monitor. Typically this will be written on the battery as a number followed by S. See the information below for more detail
```

Reply:
You are right, I think that's what I personally would prefer as a reader. I was going back and forth to make sure it's introduced before it's used like in a thesis and somehow lost the purpose.